pkg/client/subjectaccessreview.go

```diff
@@ -64,7 +64,8 @@ func (c *subjectAccessReviews) Create(sar *authorizationapi.SubjectAccessReview)
 	// if the namespace values don't match then we definitely hit an old server.  If we got a forbidden, then we might have hit an old server
 	// and should try the old endpoint
 	if (sar.Action.Namespace != result.Namespace) || kapierrors.IsForbidden(err) {
-		req, err := overrideAuth(c.token, c.r.Post().Namespace(sar.Action.Namespace).Resource("subjectAccessReviews"))
+		var req *restclient.Request
```

Comment:
same as the other change : to avoid using `:=` to initialize the vars, and so shadow the `err` var, which could then be set to `nil`